app/scripts/models/account.js

```diff
@@ -1019,13 +1019,16 @@ define(function (require, exports, module) {
     /**
      * Check whether SMS is enabled for the current account.
      *
-     * @returns {Promise} resolves to `true` if SMS is enabled,
-     *  `false` otw.
+     * @param {Object} [options] options
```

Comment:
I think so to make it consistent with https://github.com/mozilla/fxa-content-server/pull/4877#discussion_r108966702.